docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,302 @@
+
+   ```bash
+   git clone --branch v0.10.4 https://github.com/gardener/external-dns-management.git
+   git clone --branch v0.8.3 https://github.com/gardener/cert-management.git
+   ```
+
+   > **NOTE:** This tutorial uses the External DNS Management v.0.10.4 and the Certificate Management v0.8.3.
+
+3. Enable vertical Pod autoscaling on your cluster. For example, for Google Cloud Platform, run:
+
+   ```bash
+   gcloud container clusters update {CLUSTER_NAME} --enable-vertical-pod-autoscaling --zone="{CLUSTER_ZONE}" --project="{PROJECT_NAME}"
+   ```
+
+4. Install the `external-dns-management` component in the `gardener-components` Namespace:
+
+   ```bash
+   helm install external-dns {PATH_TO_EXTERNAL_DNS_MANAGEMENT}/charts/external-dns-management --namespace=gardener-components --set configuration.identifier=external-dns-identifier
+   ```
+
+5. Install the `cert-management` component in the `gardener-components` Namespace:
+
+   ```bash
+   helm install cert-manager {PATH_TO_CERT_MANAGEMENT}/charts/cert-management --namespace=gardener-components --set configuration.identifier=cert-manager-identifier
+   ```
+
+## Steps
+
+Follow these steps to set up your custom domain and prepare a certificate required to expose a service.
+
+1. Install Kyma 2.0 or higher.
+
+2. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+3. Create a Secret containing credentials for your DNS cloud service provider account. See the [official External DNS Management docuemntation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines. Remember to edit the **metadata.namespace** parameter and provide your {NAMESPACE_NAME} as the required value.  Once the YAML file with the relevant parameters is ready, create a Secret Custom Resource (CR). Run the following command:
+
+   ```bash
+   kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
+   ```
+
+4. Set up the `external-dns-management` component.
+
+- Create a DNSProvider CR. Export values of the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters as environment variables. As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`. Run:
+
+   ```bash
+   export NAMESPACE={NAMESPACE_NAME}
+   export SPEC_TYPE={PROVIDER_TYPE}
+   export SECRET={SECRET_NAME}
+   export DOMAIN={CLUSTER_DOMAIN}
```

Comment:
[Maybe not in this line, but in the description before] I think it's worth to add a note that this is the domain you're owner of. If you own "mydomain.com", then this is what you should provide here.
Later on in the tutorial you may want to expose your workload on a subdomain of this domain, e.g. "my.workload.mydomain.com", but here you should provide "mydomain.com". @mjakobczyk please confirm.